add_trailing_comma/_main.py

```diff
@@ -94,6 +94,10 @@ def main(argv: Sequence[str] | None = None) -> int:
         '--py36-plus',
         action='store_const', dest='min_version', const=(3, 6),
     )
+    parser.add_argument(
+        '--py39-plus',
+        action='store_const', dest='min_version', const=(3, 9),
+    )
```

Comment:
actually now that I look at it, I can't think of a situation where this flag actually helps.  if they're using the python3.9+ syntax already that's the only situation it'd rewrite I think ?  maybe the original way was better with the version check in the module similar to `Match`

Reply:
I think so too :+1: